blocks/library/freeform/index.js

```diff
@@ -20,30 +20,10 @@ registerBlockType( 'core/freeform', {
 	category: 'formatting',
 
 	attributes: {
-		content: children(),
+		content: prop( 'innerHTML' ),
```

Comment:
The `html` matcher is a shorthand for `prop( 'innerHTML' )`

https://github.com/aduth/hpq#api

Per effort to discourage treating as DOM node (#624), have considered whether `prop` can even be dropped.